### PR TITLE
Fix Run/Enqueue Micro tasks

### PR DESCRIPTION
**Pull request by rerobika**
- Implement missing Object::GetConstructorName

Signed-off-by: Robert Fancsik <frobert@inf.u-szeged.hu>

### CODE REVIEW

**Comment by rerobika** [comment]
Parallel status: +1409 -1071

File: deps/jerry/api.cc [hunk 1]
```diff
@@ -640,6 +640,8 @@ MaybeLocal<Value> Script::Run(Local<Context> context) {
 
   jerry_value_t result = jerry_run(script->value());
 
+  jerry_release_value(jerry_run_all_enqueued_jobs());
```
Comment (zherczeg) [inline deps/jerry/api.cc]: RunMicrotasks?

File: deps/jerry/v8jerry/v8jerry_isolate.cpp [hunk 2]
```diff
@@ -485,18 +487,18 @@ void JerryIsolate::InitalizeSlots(void) {
...
+    for (auto& task : m_micro_tasks) {
+        jerry_value_t call_res = jerry_call_function (task->value(), jerry_create_undefined(), NULL, 0);
+        jerry_release_value (call_res);
     }
-    m_tasks.clear();
+    m_micro_tasks.clear();
 
-    // TODO: is it ok to have this here?
-    jerry_run_all_enqueued_jobs();
+    jerry_release_value(jerry_run_all_enqueued_jobs());
```
Comment (zherczeg) [inline deps/jerry/v8jerry/v8jerry_isolate.cpp]: Could be a `while(true)` loop here, which breaks if no error is returned.

**Review by zherczeg — APPROVED** [review]
LGTM

**Pull request merged**